examples/gke-a3-highgpu.yaml

```diff
@@ -56,6 +56,10 @@ deployment_groups:
       master_authorized_networks:
       - cidr_block: $(vars.authorized_cidr)  # Allows your machine run kubectl command. It's required for the multi-network setup.
         display_name: "kubectl-access-network"
+      system_node_pool_taints:
+      - key: "components.gke.io/gke-managed-components"
+        value: true
+        effect: "PREFER_NO_SCHEDULE" # Allows systems like Kueue to be installed on the system default pool
```

Comment:
I suspect this will be problematic and in edge cases we will end up with workloads getting placed on the system pool. Instead of changing the taint, can we add a toleration to the kueue pods?

Reply:
I would prefer to have kueue system pod in the system NP, mainly for isolation, stability and maintenance consideration.
PREFER_NO_SCHEDULE make it possible for either case, and would be hard to maintain. Toleration for Kueue system pod seems a better fit.

Reply:
resourceFlavor might be use for workload taint/toleration setting https://kueue.sigs.k8s.io/docs/reference/kueue.v1beta1/#kueue-x-k8s-io-v1beta1-ResourceFlavorSpec

Reply:
At the moment we are downloading pod installation manifest directly from github without any modifications.
I do not want to modify that file as that has a bunch of things in there, so let me see if we can find if we can apply add tolerations after the manifest has been applied.

Reply:
I think there are two things to note:
* kueue-controller-manager which is installed via the downloaded github script, IIUC this doesn't use the ResourceFlavor based tolerations those are for workloads that are being admitted via Kueue
* Kueue workloads which in this case we might not want on the system pod as they would be for the GPU nodepool.

I agree if adding tolerations to `kueue-controller-manager` is possible that is the best way forward.

Reply:
Downloaded the manifest and added tolerations